docs/rum_collection/troubleshooting.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Troubleshooting
+kind: documentation
+further_reading:
+  - link: "https://github.com/DataDog/dd-sdk-ios"
+    tag: "Github"
+    text: "dd-sdk-ios Source code"
+  - link: "/real_user_monitoring"
+    tag: "Homepage"
+    text: "Explore Datadog RUM"
+---
+
+## Check if Datadog SDK is properly initialized
+
+After configuring Datadog SDK and running the app for the first time this is recommended to look at your debugger console in Xcode. The SDK implements several consistency checks and prints relevant warnings if something is missconfigured.
+
+## Debugging
+When writing your application, you can enable development logs by setting the `verbosityLevel` value. Relevant messages from the SDK with a priority equal to or higher than the provided level are then printed to debugger console in Xcode:
+
+```swift
+Datadog.verbosityLevel = .debug
+```
+
+If all goes well you should see output similar to this saying that a batch of RUM data was properly uploaded:
+```
+[DATADOG SDK] 🐶 → 17:23:09.849 [DEBUG] ⏳ (rum) Uploading batch...
+[DATADOG SDK] 🐶 → 17:23:10.972 [DEBUG]    → (rum) accepted, won't be retransmitted: success
+```
+
+**Note:** It is recommended to only use `Datadog.verbosityLevel` in `DEBUG` configuration and keep it unset in `RELEASE`.
+
+## Set tracking consent (GDPR compliance)
```

Comment:
Is this really "troubleshooting"? It feels like it should go on the (currently titled) Advanced Configuration page.

Reply:
All true 👌, I will move it to "Advanced Configuration" along with the other sections you suggested.